Frameworks/include/NSEnumeratorInternal.h

```diff
@@ -18,6 +18,11 @@
 
 #include <Foundation/NSEnumerator.h>
 
+#define NS_COLLECTION_THROW_ILLEGAL_KVO(KEYPATH)                                                                                         \
```

Comment:
I was looking for a shared internal place for the NS* classes where KVO is illegal.  What's the reasonable place to stick this?